### PR TITLE
Storage Move functionality wording

### PR DESCRIPTION
**Pull request by alex-deaconu**
@ezequielc @fjvallarino please review. Thanks!

### CODE REVIEW

File: src/locales/en/storage-client.po [hunk 1]
```diff
@@ -79,6 +85,21 @@ msgstr "Loading..."
 msgid "loading-functions"
 msgstr "Loading..."
 
+msgid "move.breaking-link-hide-warning"
```
Comment (ezequielc) [inline src/locales/en/storage-client.po]: @alex-deaconu Shouldn't we reuse the strings from rename warning? Thanks

**Comment by alex-deaconu** [comment]
PR updated. See comments in https://github.com/Rise-Vision/rise-vision-apps/pull/395 regarding refactoring & issues. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu 👍

**Comment by fjvallarino** [comment]
:+1:

**Pull request merged**